app/assets/stylesheets/hera/modules/_attachments.scss

```diff
@@ -38,26 +37,22 @@
     .cancel,
     .copy-link,
     .delete {
-      width: 30px;
+      width: 1.5rem;
+      text-align: right;
     }
 
     .error {
       width: 120px;
     }
 
     .filename {
-      white-space: nowrap;
+      -webkit-line-clamp: 2;
+      -webkit-box-orient: vertical;
```

Comment:
note, we are using -webkit here but this is universally adapted according to can i use. Even FF uses the -web-kit prefix for this.

Reply:
I'm sorry, it's not clear to me what you mean. It doesn't require the prefix?

Reply:
This is just an FYI for reviews wondering why we are using -webkit prefixes in production (typically features behind experimental flags are a no-no in production). In this case, the feature was released to all browsers with the -webkit prefix.